libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/FRC.kt

```diff
@@ -0,0 +1,57 @@
+package com.mapbox.navigation.base.trip.model
+
+/**
+ * Holds available [FRC] types.
+ *
+ * Available values are:
+ * - [FRC.MOTORWAY]
+ * - [FRC.TRUNK]
+ * - [FRC.PRIMARY]
+ * - [FRC.SECONDARY]
+ * - [FRC.TERTIARY]
+ * - [FRC.UNCLASSIFIED]
+ * - [FRC.RESIDENTIAL]
+ * - [FRC.SERVICE_OTHER]
+ */
+object FRC {
```

Comment:
I guess it would be nice to have a full description. It might be hard to guess the abbreviation

Reply:
What does `FRC` stand for @etl @mskurydin @SiarheiFedartsou?

Reply:
Functional road class

https://wiki.openstreetmap.org/wiki/Key:highway

  MOTORWAY
  TRUNK
  PRIMARY
  SECONDARY
  TERTIARY
  UNCLASSIFIED
  RESIDENTIAL
  SERVICE_OTHER

Reply:
https://wiki.openstreetmap.org/wiki/Key:highway I'm seeing other `highway` types under `Link roads` for example. Are we only exposing the first set? Is that on purpose? cc @etl

Reply:
Can't find `Service other` https://wiki.openstreetmap.org/wiki/Key:highway are 👇 

> I'm seeing other highway types under Link roads for example

under `SERVICE OTHER`?

Reply:
This is Mapbox classification originally based on OSM classes -> highway tag.

Reply:
So probably we shouldn't link to https://wiki.openstreetmap.org/wiki/Key:highway? 🤔 

That was my plan 😅